system/services/taskmanager/TaskManagerService.cfc

```diff
@@ -773,6 +847,46 @@ component displayName="Task Manager Service" {
 		}
 	}
 
+
+
+private numeric function _getAverageWorkTimeFromClusteredTimes( required array runTimes ) {
```

Comment:
There are quite a few methods and data structures in this function which aren't spaced by one character.
for(i=1.., [runTimes..., (ArrayAvg... etc.